src/components/PostCard/PostDetails/PostDetails.scss

```diff
@@ -54,6 +54,23 @@
   font-size: 16px;
   line-height: 24px;
   margin-bottom: 20px;
+
```

Comment:
Wondering what happens in the case of `PostCard` for these same styles? Can these styles be in `global/typography.scss`, or otherwise be reconciled there if these are now importantly specific to Post Details?

Reply:
good questions, must be @brodeur it doing but I can also look at